flows/nodes/attach.md

```diff
@@ -0,0 +1,45 @@
+# Attach
+
+The Attach node connects to an existing Gravwell query, specified by a search ID. Like the [Run Query](runquery) node, it outputs a structure into the payload (named `search` by default) which contains information about the search and allows other nodes to access the results.
```

Comment:
```suggestion
The Attach node connects to an existing Gravwell query, specified by a search ID. Like the [Run Query](runquery) node, it outputs a structure (named `search` by default) containing information about the search into the payload and allows other nodes to access the results.
```
Same comment below... I think "containing information about the search" describes the structure...?
A participial phrase should be directly next to the noun it modifies.